docs/Developers/Profile_Bears.rst

```diff
@@ -0,0 +1,74 @@
+Profile Bears
+=============
+
+This document provide the overview of coala's Profiling Interface. The
+Profiling Interface will provide an ability to Profile the Bear's code to
+optimize its performance.
+
+The profiler will start by profiling the run() method of bears because it
+consumes most of the bears time. So, this is part where bear writer will spend
+time, as rest of the part like loading the files, collecting the settings,
+etc. are done by coala itself.
+
+.. note::
+    Enabliling both profiler and debugger on a bear at a same time will result
+    into error.
+
+Bear writer will have the ability to directly dump the raw profile output
```

Comment:
writer*s*

Reply:
Since this is an "unspecified quantity" of people you are referring two, you have to choices in fact:
1. A bear writer
2. Bear writers